src/supported-operations/thin-edge-supported-operations.md

```diff
@@ -0,0 +1,127 @@
+# Supported operations
+
+## Support for operations
+
+Supported operations would be a part of thin-edge data model and supported by `operation configuration` files in the filesystem.
+
+Supported operations are listed per supported cloud.
+
+Supported operations can be modified using tedge cli tool.
+
+Operations supported by `thin-edge.io` would be stored in the `operations` directory of `thin-edge.io` configuration, it is `/etc/thin-edge/operations` by default.
+Single operation type is expressed by operation configuration file in the filesystem.
+
+Many `thin-edge.io` components can read operation configuration files and use them to perform operations, but only one component should report operations to an external connection (if supported by provider, eg. for c8y cloud it could be mapper).
+
+> Alternatives:
+
+### Option 1
+
+Operations use prefixes to identify the cloud, eg `c8y_SoftwareUpdate`, `az_SoftwareUpdate`.
+
+### Option 2
```

Comment:
I prefer Option 2. If you then also want to have the operations in the `c8y` directory start with `c8y_` because the `c8y` cloud uses that prefix for them, then you can.